install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -0,0 +1,185 @@
+#
+# Gateways Configuration
+# By default (if enabled) a pair of Ingress and Egress Gateways will be created for the mesh.
+# You can add more gateways in addition to the defaults but make sure those are uniquely named
+# and that NodePorts are not conflicting.
+# Disable specifc gateway by setting the `enabled` to false.
+#
+enabled: false
+
+istio-ingressgateway:
+  enabled: true
+  labels:
+    app: istio-ingressgateway
+    istio: ingressgateway
+  replicaCount: 1
+  autoscaleMin: 1
+  autoscaleMax: 5
+  resources: {}
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    #requests:
+    #  cpu: 1800m
+    #  memory: 256Mi
+  cpu:
+    targetAverageUtilization: 80
+  loadBalancerIP: ""
+  loadBalancerSourceRanges: {}
+  serviceAnnotations: {}
+  podAnnotations: {}
+  type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
+  #externalTrafficPolicy: Local #change to Local to preserve source IP or Cluster for default behaviour or leave commented out
+  ports:
+    ## You can add custom gateway ports
+  - port: 80
+    targetPort: 80
+    name: http2
+    nodePort: 31380
+  - port: 443
+    name: https
+    nodePort: 31390
+  # Example of a port to add
+  - port: 31400
+    name: tcp
+    nodePort: 31400
+  # Pilot and Citadel MTLS ports are enabled in gateway - but will only redirect
+  # to pilot/citadel if global.meshExpansion settings are enabled.
+  # Delete these ports if mesh expansion is not enabled, to avoid
+  # exposing unnecessary ports on the web.
+  - port: 15011
```

Comment:
is there any way by which these ports can be enabled using an option? Its very very confusing to see the list of ports open at the load balancer. Its overloaded and doesn't make for a good experience, when all the user wants is a very simple 80+443 in the load balancer.

Reply:
should be possible.  I can look into that in a separate PR.